htdocs/comm/propal/class/propal.class.php

```diff
@@ -1896,7 +1896,7 @@ public function fetch_lines($only_product = 0, $loadalsotranslation = 0, $sqlfor
 		$sql .= ' p.weight, p.weight_units, p.volume, p.volume_units,';
 		$sql .= ' d.date_start, d.date_end,';
 		$sql .= ' d.fk_multicurrency, d.multicurrency_code, d.multicurrency_subprice, d.multicurrency_total_ht, d.multicurrency_total_tva, d.multicurrency_total_ttc';
-		$sql .= ' FROM '.MAIN_DB_PREFIX.'propaldet as d';
+		$sql .= ' FROM '.MAIN_DB_PREFIX.$this->$table_element_line.' as d';
```

Comment:
there is a typo `$this->table_element_line` and not `$this->$table_element_line`